src/main/kotlin/io/hsar/wh40k/combatsimulator/logic/ActionOption.kt

```diff
@@ -21,6 +21,7 @@ import io.hsar.wh40k.combatsimulator.random.RollResult
 @JsonIgnoreProperties(value = ["actionType"])
 sealed class ActionOption {
     abstract val actionCost: ActionCost
+    abstract override fun toString(): String
```

Comment:
[Minor] You should always be able to use the class name, which would be a bit brutal but should always provide something. I recommend a default implementation.

Reply:
As with comments elsewhere in the PR ref output to user, I think a modicum of polish to make output easily readable and digestible is OK. I can change it if you want though?